### ISSUE TITLE
Inconsistent handling of kwarg with binary operator expression as value

### ISSUE THREAD

**Issue by baroquebobcat** [PyCQA/pycodestyle#673]
I think pycodestyle's handling of binary operator expressions that are the values of kwargs could be improved.

There is a case that is a little ambiguous to me as to whether it should be a style error or not. Should calls where the binary operator subexpressions are aligned with the = of the kwarg be errors? Currently, this triggers E131.
```python
call(
  kwarg=a +
        b +
        c
)
```

On the other side, dropping the lines indentation to the same as the kwarg is considered valid, which seems to me to go against the spirit of the pep8 guidelines.
```python
call(
  kwarg=a +
  b +
  c
)
```
